<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TilemapDock</class>
 <widget class="QDockWidget" name="TilemapDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Metasprite Viewer</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="labelImageView">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QGraphicsView" name="graphicsView_2"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QGraphicsView" name="graphicsView_3"/>
    </item>
    <item row="1" column="0">
     <widget class="QSpinBox" name="spinBoxRoomWidth"/>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="spinBoxRoomHeight">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
